src/jit/lsra.cpp

```diff
@@ -4537,41 +4539,45 @@ LinearScan::tryAllocateFreeReg(Interval *currentInterval, RefPosition *refPositi
     if (regType == TYP_STRUCT)
```

Comment:
I moved all of the code that checks for and handles the TYP_STRUCT case to be under #if _TARGET_ARM64_